overrides/common/style.ts

```diff
@@ -40,3 +41,35 @@ export const decorativeHeader = css`
     background: ${themeVal("color.primary")};
   }
 `;
+
+export const AccessibilityLinks = styled.a`
```

Comment:
Considering this doesn't group anything, it might make more sense to name it as singular.